code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -40,7 +40,7 @@
 		return
 	has_suit.overlays -= inv_overlay
 	has_suit = null
-	src.loc = get_turf(user || src)
+	src.forceMove(get_turf(user || src))
```

Comment:
Did you read it?

"true value if either A or B is true (neither zero, an empty string, nor null); false value otherwise"